ENIGMAsystem/SHELL/Platforms/Win32/WINDOWSfilemanip.cpp

```diff
@@ -33,6 +33,13 @@
 #include <time.h>
 #include <sys/stat.h>
 
+#include <fstream>
+#include <algorithm>
+#include <direct.h>
+#include <dirent.h>
```

Comment:
These are POSIX headers... they aren't guaranteed to exist on Windows.

Reply:
Well, they come with MinGW, and that's what ENIGMA uses to compile on Windows. Or does that not matter? i.e. the header may try to implement features that won't be recognized on some Windows systems?